### DISCUSSION TITLE
Translation help

### DISCUSSION THREAD

**Discussion by M1ck0** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1181] [Q&A] [answered]
I want to help translate into my language. Where do I upload the translation file after finishing the translation?

I already have bought premium so I just wanna do this for the community, not the discount.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @M1ck0,

Thank you for your contribution.
We have a detailed guide on our website "[Translate AlDente](https://apphousekitchen.com/translate-aldente-and-get-50-off-aldente-pro-for-a-year/)". Alternatively, you can also just upload the new translation here on GitHub.

Best,
Matthias
